test/systemtest/README.md

```diff
@@ -0,0 +1,44 @@
+# OpenTitan System Tests
+
+System tests are end-to-end tests for the whole OpenTitan system. They operate
+on build outputs (typically in `build-bin`) and can be used to check that the
+full system, consisting of software, hardware, and tooling, works as expected.
+
+Tests can be executed against different target platforms (such as FPGAs or
+simulations), even though not all OpenTitan-supported targets are supported at
+the moment.
+
+## Available tests
+
+* `earlgrey/test_sim_verilator.py`: Run various software tests against a
+  Verilator-built simulation of an Earl Grey design.
+
+## Run the tests
+
+### Prerequisites
+
+Before the tests can be run, the various components of OpenTitan need to be
+built and present in the `BIN_DIR` directory (typically `$REPO_TOP/build-bin`).
+
+Alternatively, the tests can be executed against any pre-built OpenTitan
+snapshot available from GitHub releases, or from CI.
+
+### Test execution
+
+The pytest documentation gives a good overview of the available options to run
+tests in the [Usage and Invocations](https://docs.pytest.org/en/stable/usage.html)
+chapter. Here's a quick reference.
+
+```sh
+# Run all tests
+pytest test/systemtest
+
+# Run tests in a specific file
+pytest test/systemtest/earlgrey/test_sim_verilator.py
+
+# Run a specific test in a specific parametrization
+pytest test/systemtest/earlgrey/test_sim_verilator.py -k test_apps_selfchecking[usbdev_test]
```

Comment:
Can you clarify how this works? If I want to use the rest runner with some target, will that just work, or does it have to be in the list given in the other file?

Reply:
By "target" you mean a piece of software? They are listed in test/systemtest/config.py, which is then made available as parameter to the test_apps_selfchecking test.